package.json

```diff
@@ -26,9 +26,9 @@
     "dev": "cross-env NODE_ENV=development gulp dev",
     "test": "jest",
     "test:watch": "jest --watch",
-    "lint": "npx eslint \"{src,test,scripts}/**/*.{js,jsx,ts,tsx}\"",
-    "lint:fix": "npx eslint --fix \"{src,test,scripts}/**/*.{js,jsx,ts,tsx}\"",
-    "extract": "formatjs extract \"src/**/*.{js,jsx,ts,tsx}\" --out-file temp.json --flatten --id-interpolation-pattern '[sha512:contenthash:base64:6]' --preserve-whitespace",
+    "lint": "tsc --noEmit && eslint \"{src,test,scripts}/**/*.{js,jsx,ts,tsx}\"",
```

Comment:
Isn't this `tsc --noEmit` also required for the next line?

Reply:
No, the <b>error</b> `tsc --noEmit` brings need to be manually addressed

<b>error</b> - type error as per tsconfig